foundations/installations/setting_up_git.md

```diff
@@ -187,4 +187,5 @@ You probably felt like you were way in over your head, and you probably didn't u
 
 This section contains helpful links to related content. It isn’t required, so consider it supplemental.
 
-*   It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum.
+*   [Understanding SSH Key Pairs](https://winscp.net/eng/docs/ssh_keys) SSH is an implimintation of public-key cryptography, also known as asymmetric cryptography. Having basic understanding of how it works can help you understand what SSH key is about.
```

Comment:
A couple of grammatical fixes needed here. Otherwise it looks good.

Ideally, we should probably link to a less Windows focused source, considering we don't currently support the OS within our curriculum. Perhaps https://www.ssh.com/academy/ssh/key ?

```suggestion
*   [Understanding SSH Key Pairs](https://winscp.net/eng/docs/ssh_keys) SSH is a secure network protocol that uses an implementation of public-key cryptography, also known as asymmetric cryptography. Having a basic understanding of how it works can help you understand what an SSH key is all about.
```